xskillscore/core/probabilistic.py

```diff
@@ -311,17 +322,32 @@ def brier_score(observations, forecasts, dim=None, weights=None, keep_attrs=Fals
     * Brier, Glenn W. "VERIFICATION OF FORECASTS EXPRESSED IN TERMS OF PROBABILITY."
       Monthly Weather Review, 78(1): 1-3
       https://journals.ametsoc.org/doi/abs/10.1175/1520-0493%281950%29078%3C0001%3AVOFEIT%3E2.0.CO%3B2
-
+    * C. A. T. Ferro. Fair scores for ensemble forecasts. Q.R.J. Meteorol. Soc., 140:
+      1917–1923, 2013. doi: 10.1002/qj.2270.
+    * https://www-miklip.dkrz.de/about/problems/
     """
-    res = xr.apply_ufunc(
-        properscoring.brier_score,
-        observations,
-        forecasts,
-        input_core_dims=[[], []],
-        dask="parallelized",
-        output_dtypes=[float],
-        keep_attrs=keep_attrs,
-    )
+    if fair:
+        if member_dim not in forecasts.dims:
+            raise ValueError(
+                "need forecast with member dim"
+            )  # TODO: check that brier_score always get member in forecast
+        else:
+            M = forecasts[member_dim].size
+            e = (forecasts == 1).sum(member_dim, keep_attrs=keep_attrs)
+            o = observations
+        with xr.set_options(keep_attrs=keep_attrs):
```

Comment:
can this `with xr.set_options(keep_attrs=keep_attrs)` be set higher?

```
M = forecasts[member_dim].size
o = observations
with xr.set_options(keep_attrs=keep_attrs):
    e = (forecasts == 1).sum(member_dim)
    res = (e / M - o) ** 2 - e * (M - e) / (M ** 2 * (M - 1))
```
In fact is that needed if you do `res.attrs = observations.attrs` later?